internal/server/telemetry.go

```diff
@@ -90,7 +90,28 @@ func (t *Telemetry) EnqueueHeartbeat() {
 		logging.Debugf("%s", err.Error())
 	}
 
-	t.Event("heartbeat", "", map[string]interface{}{
+	first, err := data.ListIdentities(t.db, &models.Pagination{Limit: 1}, data.NotName(models.InternalInfraConnectorIdentityName))
+	if err != nil {
+		logging.Debugf("%s", err.Error())
+	}
+
+	var admin models.Identity
+	if len(first) > 0 {
+		admin = first[0]
```

Comment:
Is this guaranteed to be the admin? What if Infra was setup using a config file?

Reply:
The list of `ListIdentities` is most likely alphabetical as well, so probably not the admin.

I think we likely need to query for grants and find the `infra/admin` grant with the earliest created time.

Reply:
Thanks for the review. This should be ready for another quick look. @BruceMacD @dnephin